webcompat/templates/cssfixme.html

```diff
@@ -20,7 +28,13 @@
           <button type="button" class="Button Button--default" id="btn_do_fixup">Add standard equivalents for prefixed CSS</button>
         </div>
         <div class="wc-CSSFixme-item">
-          <pre class="wc-CSSFixme-resultat" id="fixedcss"></pre>
+          <div id="js-CSSFixme"></div>
```

Comment:
@hallvors I removed `<pre>`  element and add this parent. So you can add <pre> directly in this div. if you can add a class : `class="wc-CSSFixme-resultat"` would be awesome ;)